codegen/server/capi_repo.ts

```diff
@@ -0,0 +1,96 @@
+import { PermanentMemo, TimedMemo } from "../../util/memo.ts"
+import { shaAbbrevLength } from "./git_utils.ts"
+import { CodegenServer } from "./server.ts"
+
+const tagsTtl = 60_000 // 1 minute
```

Comment:
Nit: any reason to deviate from constant naming convention of JS?

Reply:
Life's too short for typing in SCREAMING_SNAKE_CASE.

Just personal preference; do you want me to change it?